banzai/main.py

```diff
@@ -361,6 +366,11 @@ def run_preview_pipeline():
     args.preview_mode = True
     args.raw_path = None
     args.filename = None
+
+    logs.set_log_level(args.log_level)
+    # Need to keep the amqp logger level low, or else it send heartbeat check messages every second
+    logging.getLogger('amqp').setLevel('INFO')
```

Comment:
Shouldn't this be set to args.log_level?

Reply:
No, because if the `amqp` log level is `DEBUG`, then it will log a heartbeat check every second for each listener. 
Maybe instead I could set the log level to be max(INFO, args.log_level)?